bids/layout/tests/test_validation.py

```diff
@@ -845,53 +673,3 @@ def test_is_subject_false(testvalidator):
 #     for item in target_list:
 #         result = testvalidator.is_field_map(item)
 #         assert not result
-
-
-# checks is_phenotypic() function true cases
-def test_is_phenotypic_true(testvalidator):
-    target_list = [
-        "/phenotype/measurement_tool_name.tsv",
-        "/phenotype/measurement_tool_name.json"
-    ]
-
-    for item in target_list:
-        result = testvalidator.is_phenotypic(item)
-        assert result
-
-
-# checks is_phenotypic() function true cases
-def test_is_phenotypic_false(testvalidator):
-    target_list = [
-        "/measurement_tool_name.tsv",  # missed phenotype dir
-        "/phentype/measurement_tool_name.josn"  # wrong phenotype dir
-        "/phenotype/measurement_tool_name.jsn"  # wrong extension
-    ]
-
-    for item in target_list:
-        result = testvalidator.is_phenotypic(item)
-        assert not result
-
-
-def test_index_associated_false(testvalidator):
-    testvalidator = BIDSValidator(index_associated=False)
-    target_list = [
-        "/code/",
-        "/derivatives/",
-        "/sourcedata/",
-        "/stimuli/",
-        "/.git/"
-    ]
-
-    for item in target_list:
-        result = testvalidator.is_associated_data(item)
-        assert not result
-
-def test_layout_with_validation():
-    data_dir = join(get_test_data_path(), '7t_trt')
-    layout1 = BIDSLayout(data_dir, validate=True)
-    layout2 = BIDSLayout(data_dir, validate=False)
-    assert len(layout1.files) < len(layout2.files)
-    # Not a valid BIDS file
-    badfile = join(data_dir, 'test.bval')
-    assert badfile not in layout1.files
-    assert badfile in layout2.files
```

Comment:
Unless this test exists elsewhere, it's worth keeping.

Reply:
ah yes true

good catch